user-service/src/main/java/hcmus/alumni/userservice/controller/UserServiceController.java

```diff
@@ -492,4 +534,530 @@ public ResponseEntity<HashMap<String, Object>> getUser(@PathVariable String id)
 		result.put("user", user.get());
 		return ResponseEntity.status(HttpStatus.OK).body(result);
 	}
+
+	@GetMapping("/{id}/profile")
+	public ResponseEntity<Map<String, Object>> getProfileInfo(@PathVariable String id) {
+		Optional<IUserProfileDto> optionalUser = userRepository.findUserProfileById(id);
+
+		if (optionalUser.isEmpty()) {
+			throw new AppException(21200, "Không tìm thấy thông tin người dùng", HttpStatus.NOT_FOUND);
+		}
+
+		IUserProfileDto user = optionalUser.get();
+		Optional<IAlumniProfileDto> optionalAlumni = alumniRepository.findAlumniProfileById(id);
+
+		Map<String, Object> response = new HashMap<>();
+
+		Optional<IVerifyAlumniProfileDto> alumniVerificationOptional = verifyAlumniRepository
+				.findByUserIdAndIsDelete(id, false);
+
+		response.put("user", user);
+		response.put("alumni", optionalAlumni.orElse(null));
+		response.put("alumniVerification", alumniVerificationOptional.orElse(null));
+
+		return ResponseEntity.status(HttpStatus.OK).body(response);
+	}
+
+	@PutMapping("/profile")
+	public ResponseEntity<String> updateBasicProfileInfo(@RequestHeader("userId") String userId,
+			@RequestBody ProfileRequestDto profileRequestDto) {
+		Optional<UserModel> optionalUser = userRepository.findById(userId);
+
+		if (optionalUser.isEmpty()) {
+			throw new AppException(21300, "Không tìm thấy thông tin người dùng", HttpStatus.NOT_FOUND);
+		}
+
+		UserModel user = optionalUser.get();
+		UserDto userDto = profileRequestDto.getUser();
+		AlumniDto alumniDto = profileRequestDto.getAlumni();
+		if (userDto != null) {
+			if (userDto.getFullName() != null) {
+				user.setFullName(userDto.getFullName());
+			}
+			if (userDto.getFacultyId() != null) {
+				user.setFaculty(new FacultyModel(userDto.getFacultyId()));
+			}
+			if (userDto.getSexId() != null) {
+				user.setSex(new SexModel(userDto.getSexId()));
+			}
+			if (userDto.getSocialMediaLink() != null) {
+				user.setSocialMediaLink(userDto.getSocialMediaLink());
+			}
+			if (userDto.getPhone() != null) {
+				user.setPhone(userDto.getPhone());
+			}
+			if (userDto.getAboutMe() != null) {
+				user.setAboutMe(userDto.getAboutMe());
+			}
+		}
+
+		Optional<AlumniModel> optionalAlumni = alumniRepository.findByUserId(userId);
+		if (optionalAlumni.isPresent() && alumniDto != null) {
+			AlumniModel alumni = optionalAlumni.get();
+			if (alumniDto.getAlumClass() != null) {
+				alumni.setAlumClass(alumniDto.getAlumClass());
+			}
+			if (alumniDto.getGraduationYear() != null) {
+				alumni.setGraduationYear(alumniDto.getGraduationYear());
+			}
+			alumniRepository.save(alumni);
+		}
+		userRepository.save(user);
+		return ResponseEntity.status(HttpStatus.OK).body("");
+	}
+
+	@PreAuthorize("hasAnyAuthority('User.Edit')")
+	@PutMapping("/alumni-verification/{id}")
```

Comment:
```suggestion
	@PutMapping("/alumni-verification/{id}")
```
`{id}` ở đây là `userId` hay `id` của cái `verify_alumni` v
nếu là `userId` thì có thể lược bỏ